test/worlds/graded_buoyancy.sdf

```diff
@@ -33,6 +33,15 @@
           <density>1</density>
         </density_change>
       </graded_buoyancy>
+
+      <!-- whitelist by link name -->
+      <white_list>lighter_than_water::ball::body</white_list>
```

Comment:
`whilte_list` -> `enable`

Should the test be failing without this change?

Reply:
Ah good to catch this, the `no_buoyancy` model was too dense and would sync either way. Fixed in 5b9bc46